marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/test-data/mappings/OrdersMapping/OrdersMapping-4.mapping.json

```diff
@@ -0,0 +1,37 @@
+{
+  "lang": "zxx",
+  "name": "OrdersMapping",
+  "description": "The structured properties are mapped to invalid properties",
+  "version": 4,
+  "targetEntityType": "http://marklogic.com/data-hub/example/OrderType-0.0.1/OrderType",
+  "sourceContext": "/",
+  "properties": {
+    "id": {
+      "sourcedFrom": "id"
+    },
+    "purchaseDate": {
+      "sourcedFrom": "parseDate(transactionDate, 'MM/DD/YYYY')"
+    },
+    "orderCost": {
+      "sourcedFrom": "sum(items ! (quantity * price))"
+    },
+    "customer": {
+      "targetEntityType": "http://marklogic.com/data-hub/example/CustomerType-0.0.1/CustomerType",
+      "sourcedFrom": "customer",
+      "properties": {
+        "firstname": {
+          "sourcedFrom": "invalidFirstName"
+        }
+      },
+      "items": {
```

Comment:
This doesn't look right - shouldn't "items" either be under "customer/properties" or at the same level as "customer"?